MIGRATION_GUIDE.md

```diff
@@ -4,6 +4,11 @@ This document is meant to help you migrate your Terraform config to the new newe
 describe deprecations or breaking changes and help you to change your configuration to keep the same (or similar) behavior
 across different versions.
 
+## v0.94.0 ➞ v0.94.1
+### *(breaking change)* changes in snowflake_schema
```

Comment:
It is not a breaking change IMO

Reply:
IMO it is - we're changing current behavior so that objects are not dropped.

Reply:
And how this is breaking anything from the user's perspective? And if you want it to be a breaking, you cannot release it as bugfix. I would say, this is an implementation detail in this case.

Reply:
Fixed.